products/models.py

```diff
@@ -0,0 +1,20 @@
+from django.db import models
+from status.models import StatusGeneral
+
+class Product(models.Model):
+    id = models.AutoField(primary_key=True) 
+    name = models.CharField(max_length=30)
+    description = models.CharField(max_length=50)
+    price = models.DecimalField(max_digits=11, decimal_places=2)
+    status = models.ForeignKey(StatusGeneral, on_delete=models.CASCADE)
```

Comment:
`major`: attention with on_delete CASCADE. This configuration will remove all related records.